services/harmony/app/util/parameter-parsers.ts

```diff
@@ -79,10 +100,23 @@ export function handleCrs(
  */
 export function handleScaleExtent(
   operation: DataOperation,
-  query: Record<string, string>): void {
+  query: Record<string, number[] | string>): void {
   if (query.scaleextent) {
-    const [xMin, yMin, xMax, yMax] = query.scaleextent;
-    operation.scaleExtent = { x: { min: xMin, max: xMax }, y: { min: yMin, max: yMax } };
+    try {
+      if (typeof query.scaleextent === 'string') {
+        const scaleExtentString = query.scaleextent.replace('(', '').replace(')', '');
+        const [xMinString, yMinString, xMaxString, yMaxString] = scaleExtentString.split(/,\s*/);
```

Comment:
nit: I think it can be slightly more precise:
```
const [xMin, yMin, xMax, yMax] = scaleExtentString.split(/,\s*/).map(parseNumber);
operation.scaleExtent = {
          x: { min: xMin, max: xMax },
          y: { min: yMin, max: yMax },
        };
```